src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CashoutAggregate.cs

```diff
@@ -182,7 +182,7 @@ public bool OnOperationFailed(string error)
 
             Error = error;
 
-            Result = CashoutResult.Failure;
+            Result = cashoutResult;
```

Comment:
Add error code property to aggregate.
Pass here CashoutResult.Failure and error code as separate nullable enum.

Reply:
I suggest to add  new ErrorCode nullable enum to Aggregate. Values - nearly the same as ErrorCode in CashoutFailedEvent